### ISSUE TITLE
Integrate new map shader for map and minimap

### ISSUE THREAD

**Issue by davidejensen** [decentraland/unity-renderer#2652]
Based on what has been created in issue #2353 integrate the shader for the map and minimap.

Tasklist:
- [ ] Integrate shader for map
- [ ] Integrate shader for minimap
- [ ] Adapt zoom and move behavior in the map at a shader level
- [ ] Adapt the centering functionality to a shader level
- [ ] Adapt the overlay system to the shader map

**Comment by JorgeDecentraland** [comment]
blocked until @olavra and @JuanSalice coordinate in order to validate the current implementation

**Comment by olavra** [comment]
- [ ] More subtle 1x1 divisions
- [ ] Wrong data for districts or plazas (like Dragon City)
- [ ] Some black spots in the corners
- [ ] Grid + borders should behave properly depending on the zoom (mipmap?)
- [ ] rainbow in the border should work with masked UI
- [ ] Border inside districts and plazas should be of a brighter tone of the parcel base color

**Comment by aixaCode** [comment]
https://api.decentraland.org/v1/estatemap.png

**Comment by xavi-dcl** [comment]
@AjimenezDCL what needs to happen to get this one unblock?

**Comment by m3taphysics** [comment]
Closing in favour of #4950
